hugo/layouts/partials/site/head.html

```diff
@@ -1,5 +1,6 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta name="description" content="{{ with .Description | $.RenderString | plainify }}{{ . }}{{ else }}{{ with .Summary }}{{ . | plainify | htmlUnescape | chomp | truncate 160 -}}{{ else }}{{ with .Site.Params.description }}{{ . | $.RenderString | plainify }}{{ end }}{{ end }}{{ end }}">
```

Comment:
Maybe make this multiple lines? It's a bit hard to read and exceeds the max line length

Reply:
yes i agree